.lighthouserc.js

```diff
@@ -0,0 +1,234 @@
+              'uses-passive-event-listeners': ['error', {minScore: 1}],
+              'no-document-write': ['error', {minScore: 1}],
+              'external-anchors-use-rel-noopener': ['error', {minScore: 1}],
+              'geolocation-on-start': ['error', {minScore: 1}],
+              'doctype': ['error', {minScore: 1}],
+              'no-vulnerable-libraries': ['error', {minScore: 1}],
+              'js-libraries': ['error', {minScore: 1}],
+              'notification-on-start': ['error', {minScore: 1}],
+              'deprecations': ['error', {minScore: 1}],
+              'password-inputs-can-be-pasted-into': ['error', {minScore: 1}],
+              // 'errors-in-console': ['warn', {minScore: 1}], //failing
+              'image-aspect-ratio': ['error', {minScore: 1}]
+            }
+          },
+          {
+            matchingUrlPattern: 'http://[^/]+/library',
+            assertions: {
+               // Performance Opportunities
+              'render-blocking-resources': ['warn', {minScore: 0}], // failing
+              'unminified-css': ['warn', {minScore: 0}], // failing 
+              'unminified-javascript': ['warn', {minScore: 0}], // failing
+              'unused-css-rules': ['warn', {minScore: 0}], // failing
+              'uses-text-compression': ['warn', {minScore: 0}], // failing
+              // Best Practice Audits
+              'uses-http2': ['error', {minScore: 0}], // failing
+              'errors-in-console': ['error', {minScore: 0}], //failing
+            }
+          },
+          {
+            matchingUrlPattern: 'http://[^/]+/get_started',
+            assertions: {
+               // Performance Opportunities
+               'render-blocking-resources': ['warn', {minScore: 0}], // failing
+               'unminified-css': ['warn', {minScore: 0}], // failing 
+               'unminified-javascript': ['warn', {minScore: 0}], // failing
+               'unused-css-rules': ['warn', {minScore: 0}], // failing
+               'uses-text-compression': ['warn', {minScore: 0}], // failing
+               // Best Practice Audits
+               'uses-http2': ['error', {minScore: 0}], // failing
+               'errors-in-console': ['error', {minScore: 0}], //failing
+  
+            }
+          },
+          {
+            matchingUrlPattern: 'http://[^/]+/donate',
+            assertions: {
+               // Performance Opportunities
+               'render-blocking-resources': ['warn', {minScore: 0}], // failing
+               'unminified-css': ['warn', {minScore: 0}], // failing 
+               'unminified-javascript': ['warn', {minScore: 0}], // failing
+               'unused-css-rules': ['warn', {minScore: 0}], // failing
+               'uses-text-compression': ['warn', {minScore: 0}], // failing
+               // Best Practice Audits
+               'uses-http2': ['error', {minScore: 0}], // failing
+               'errors-in-console': ['error', {minScore: 0}], //failing
```

Comment:
Why are these two failing?

Reply:
I chose to comment "failing" on tests that aren't at my score: 1, and we remove the comments once they are no longer failing? Maybe this is unnecessary,

Reply:
I'm asking why these two assertions don't pass, are there some errors in the console, if yes what errors? And I assume the HTTP2 is not used on the dev server that is why the assertion is failing.

Reply:
Also, if they fail in all pages, you can probably move them to the "default" config part.